crates/common/log_manager/src/error.rs

```diff
@@ -20,6 +20,6 @@ pub enum LogRetrievalError {
     #[error("Log file has maximum number of lines.")]
     MaxLines,
 
-    #[error("No such file or directory for log type: {log_type}")]
+    #[error("No logs found with type {log_type:?}")]
```

Comment:
I've changed this to your suggestion now, without the colon since the quotes provide delimiting. At the moment, the code doesn't handle these quotes correctly when sending SmartREST messages, but I'll address that separately since fixing that breaks a lot of tests that rely on needlessly quoted values in CSV files.

Reply:
> At the moment, the code doesn't handle these quotes correctly when sending SmartREST messages, but I'll address that separately since fixing that breaks a lot of tests that rely on needlessly quoted values in CSV files.

Yeah, blindly quoting all SmartREST values, even for the ones that don't need quoting, has always been a pain while writing tests. I'd be thanking you, if you can fix that.